changelog/fragments/2852-pr.yaml

```diff
@@ -0,0 +1,34 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated and it will no longer scaffold the `/bin/ao-logs` which was using it to print the Ansible logs in the side-car since the side-car ansible container was removed in the previous versions. [#2852](https://github.com/operator-framework/operator-sdk/pull/2852)
+
+# kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "removal"
+
+    # Is this a breaking change?
+    breaking: true
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: The `inotify-tools` dependency was removed and `/bin/ao-logs` are no longer generated for Ansible based-operators
```

Comment:
Nit: not a huge deal, but this is probably a little long for a header and might not render super well in the migration guide.

Not sure what the best general approach is for this. We could add a header character limit that we check when validating as part of the sanity test. WDYT?